linode/manager
# upcoming: [M3-7294] - AGLB Full Create Flow - Add Rule Support to Routes

## Description 📝

- Adds rule support for AGLB routes in the full create flow

> [!warning]
> I'm putting this PR up for review as-is. There are plans to change the POST payload, which will require some changes to how this flow works. I want to merge this PR before we pause work on the create flow so the work isn't lost. 

## Preview 📷

| Before  | After  |
|---|---|
| ![Screenshot 2024-01-16 at 4 46 36 PM](https://github.com/linode/manager/assets/115251059/c8d84d7f-0992-479b-9889-2d9254061384)  | ![Screenshot 2024-01-16 at 4 45 58 PM](https://github.com/linode/manager/assets/115251059/5e124b22-3f47-4a74-a6bd-c62a24e0eaa7)  |

## How to test 🧪

- Check out this PR
- Go to http://localhost:3000/loadbalancers/create
- Turn on the `AGLB Full Create Flow` feature flag
- Verify you can **create, edit, and delete** an AGLB Route **rule** in the create flow

> [!note]
> We will write automated testing for this flow after the fact because of it's complexity and because the API spec may be changing soon

## As an Author I have considered 🤔
